contents/handbook/strategy/strategy.md

```diff
@@ -170,3 +170,111 @@ To get 5 reference customers in the controlled data / growth product segment we
 **Two Word Version**
 
 “Nail Funnels”
+
+### Milestone 2 (Eary August Onwards)
+
+After nailing funnels, our primary focus should transition to Diagnosing causes, since our customers will now be able to understand when problems are happening with their nailed funnels but to address these issues they must go one step deeper and understand why they are occurring.
+
+In parallel we should spin-up evergreen investments around growth and our core platform (Performance, Reliability and Extensibility) to ensure we are reaching as many organizations as possible (increasing the number of successful products in the world) and so we don’t lose trust with existing users due to issues underpinning our platform.
+
+**Once Sentence Version**
+Nailing funnels means people will find problems with their products, next we need to enable them to Diagnose Causes so they can fix these problems and make their product successful.
```

Comment:
This is clunky - reads like two sentences that have been mashed together with a sneaky comma.